sample_rules/basic_rules_atomic.yaml

```diff
@@ -128,7 +128,7 @@
             - "_clear_packaging_caches_and_temporary_package_downloads"
         - 
           label: "yum_update_all"
-          regex: /yum(.+update all|.+upgrade|.+update|\.config)/
+          regex: /yum([ t]+update all|[ t]+upgrade|[ t]+update|\.config)/
```

Comment:
I think every `t` needs escaping, like this: `[ \t]`.

Reply:
The code has been updated to account for `[ \t]` instead.  Thanks!